xinference/api/restful_api.py

```diff
@@ -808,6 +828,60 @@ async def build_gradio_interface(
 
         return JSONResponse(content={"model_uid": model_uid})
 
+    async def build_gradio_images_interface(
+        self, model_uid: str, request: Request
+    ) -> JSONResponse:
+        """
+        Build a Gradio interface for image processing models.
+        """
+        payload = await request.json()
+        try:
+            body = BuildGradioImageInterfaceRequest.parse_obj(payload)
+        except ValidationError as e:
+            logger.info(e.json())
```

Comment:
This error should be raised, and it must be logger.error not logger.info

Reply:
Yes, I will remove unnecessary loggers.